docs/README.md

```diff
@@ -1,55 +1,59 @@
-* **[Avatar](generated_docs/controller/Avatar.md)** - *This is the public facing part of the DAO that handles the interaction of the DAO with the rest of the world(interacing with other DAOs, paying third party participants, etc...)*.
-* **Subscribed [Elements](#elements)** - *The set of [Schemes](generated_docs/universalSchemes) and [Global Constraints](generated_docs/globalConstraints) the DAO consists of*.
+![The DAO stack](img/controller.png)
 
-Those components are organized in a single smart contract called a *[Controller](Controller)*.
+* **[Native token](generated_docs/controller/DAOToken.md)** - *A Standard [ERC20 token](https://theethereum.wiki/w/index.php/ERC20_Token_Standard) that defines the value system of the DAO. Participants are rewarded for their contributions based on this token*.
+* **[Reputation](generated_docs/controller/Reputation.md) system** - *Similar to native token but is **non-transferable**. The reputation system is used to assign weight the votes of participants*.
+* **[Avatar](generated_docs/controller/Avatar.md)** - *This is the public-facing part of the DAO that handles the interaction of the DAO with the rest of the world(eg. interacing with other DAOs, paying third party participants, etc...)*.
+* **Subscribed [Elements](#elements)** - *The set of [Schemes](generated_docs/universalSchemes) and [Global Constraints](generated_docs/globalConstraints) that comprise the DAO*.
+
+For each DAO, the above components are organized in a single smart contract called a *[Controller](generated_docs/controller/Controller)*, which acts as the owner of the other components.
 
 ## Elements
 
 The main configurable and extendable building blocks of DAOs are:
 *Schemes* and *Global Constraints*, which are collectively called *elements*.
 
-* **[Schemes](generated_docs/universalSchemes/UniversalSchemeInterface.md)** *define the "do's" of the DAO, defining rules under which actions are taken, based on the participants input.* Examples of Schemes are:
-    * Rewarding contributions if other participants recognize them as worthy.
+* **[Schemes](generated_docs/universalSchemes/UniversalSchemeInterface.md)** *define the "dos" of the DAO, the rules under which actions are taken, based on the participants' input.* Examples of Schemes are:
+    * Rewarding contributions when other participants recognize them as worthy.
     * Voting on proposals and automatically executing the chosen proposal.
-* **[Global constraints](generated_docs/globalConstraints/GlobalConstraintInterface.md)** *define the "don't" of the DAO, they define conditions that must always hold and can prevent an action from being taken if it violates these conditions.* Examples of Global Constraints are:
+* **[Global constraints](generated_docs/globalConstraints/GlobalConstraintInterface.md)** *define the "don'ts" of the DAO, the conditions that must always hold and can prevent an action from being taken if it violates these conditions.* Examples of Global Constraints are:
```

Comment:
should this link be replaced with a link to contracts/globalConstraints/README.md ?
which should give details explanation of globalConstraints?
Same for controller and voting machines.

Reply:
updated at: https://github.com/daostack/arc/pull/309/commits/ec72cd04e458beaad55e670c756d3a254a6be84c